### PR TITLE
Add output dpms to manpage

### PR DESCRIPTION
**Pull request by emersion**

### CODE REVIEW

**Comment by bendooru** [comment]
I think the description is backwards: `output * dpms enabled` will turn my screens on whereas `output * dpms disabled` will turn them off.

**Comment by Emantor** [comment]
I use `output * dpms off` and `output * dpms on` which turn the output off and on.

**Comment by emersion** [comment]
Updated

File: sway/sway-output.5.scd [hunk 1]
```diff
@@ -80,14 +79,18 @@ must be separated by one space. For example:
...
+	_anticlockwise_) then the transform is added or subtracted from the current
+	transform.
 
 *output* <name> disable|enable
 	Enables or disables the specified output (all outputs are enabled by
 	default).
 
+*output* <name> dpms enabled|disabled
+	Enables or disables the specified output via DPMS. To turn an output off
+	(ie. blank teh screen but keep workspaces as-is), one can DPMS disable it.
```
Comment (RedSoxFan) [inline sway/sway-output.5.scd]: s/teh/the/

Comment (emersion) [reply sway/sway-output.5.scd]: Damn, fixed

**Comment by RedSoxFan** [comment]
Since the default config uses `dpms on` and `dpms off` with the swayidle example, should this be changed to `dpms on|off` for consistency?

**Comment by emersion** [comment]
Right, updated

**Comment by RedSoxFan** [comment]
Thanks!

**Pull request merged**